gimpopenvino/tools/stable-diffusion-ov-server.py

```diff
@@ -113,89 +124,106 @@ def run(model_name,device_name):
 
     log.info('Initializing Inference Engine...')
     log.info('Model Path: %s',model_path )
-    log.info('device_name: %s',device_name)
+    log.info('supported device_name: %s',device_name)
```

Comment:
device name is no longer an array, so let's remove this.

Reply:
Done